src/index.ts

```diff
@@ -99,15 +110,40 @@ export class DrupalJsonApiParams {
     return key;
   }
 
-  public getQueryString(): string {
+  public mergeQueryObject(params: Params): DrupalJsonApiParams {
```

Comment:
I wanted to include such functionality. But after introducing support for `optimised` query, it became difficult to create various test cases. Hence, looked away.